robot_length: 2.1
robot_width: 1.0
safety_dist: 0.1
clear_tolerance: 10
clear_radius: 1.1

# limits
forward_limit: 1.0
horizontal_limit: 10.0

# goal path
delta_theta: 20 # degrees
wp_interval: 10 # number of wp in between each goal
pose_tolerance: 10 # adjust if pose deviates more than this
goal_stop: 1.0 # reached goal if within this distance from goal

# run mode
operation: false # false: not running with robot
icr: false # false: does not search for optimal icr

# ultrasonic sensor
side_spacing: 0.3

# speeds
factor: 1.5
vx: 0.1 # max speed = vx * factor
wz: 0.05
